app/models/rook.rb

```diff
@@ -1,2 +1,6 @@
 class Rook < Piece
+  def piece_rules?(to_x:, to_y:)
+    return true if to_y == y_position || to_x == x_position
```

Comment:
This is ready to be merged now. Jack and I went through the move logic and everything is okay. We debated whether or not to include multiple rspec tests for both left and right horizontal moves and up and down vertical moves and decided to include them for now. On our next call with Chorn, we should ask about how thorough we need to be in writing tests.